src/components/content-tab-import-export.tsx

```diff
@@ -21,13 +21,17 @@ interface ContentTabImportExportProps {
 	selectedSite: SiteDetails;
 }
 
+const siteSyncingMessage = () =>
+	__( 'This site is being synced. Please wait for the sync to finish before you export it.' );
```

Comment:
Yeah, to be honest, I am not very satisfied with having that translation in the function either. I was trying to find some simple way that doesn't add too much overhead (just to get that single string translated).

Now with the changes needed to address the other bug you noticed (tooltip message on the "Drop file" area), we will now need both strings shared across both components. I will continue looking into this tomorrow with fresh brain. 🙂

Reply:
Turns out that the changes won't be necessary because each string should be different anyways. In other words, there is no need to use any of the strings in multiple components.